app/models/user.rb

```diff
@@ -112,4 +112,9 @@ def has_live_cases?
   def multiple_team_member?
     team_roles.size > 1
   end
+
+  def can_link_case?(kase)
+    return true unless kase.is_a?(Case::SAR)
```

Comment:
I think we should be doing this check in the `SARPolicy`, otherwise we'll have 3 places to configure this: `BasePolicy`, `User` and here.

Reply:
OK, fair enough.